.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/ @gomods/contributors @gomods/maintainers
+/pkg @gomods/contributors
```

Comment:
the rule for a directory _and_ it's contents should be `/pkg/`, if that's what your going for.